internal/hardware/validator.go

```diff
@@ -27,11 +27,11 @@ import (
 )
 
 const (
-	tooSmallDiskTemplate       = "Disk is too small (disk only has %s, but %s are required)"
-	wrongDriveTypeTemplate     = "Drive type is %s, it must be one of %s."
-	wrongMultipathTypeTemplate = "Multipath device has path of type %s, it must be %s"
-	iSCSIWithMultipathHolder   = "iSCSI disk with a multipath holder is not eligible"
-	wrongISCSINetworkTemplate  = "iSCSI host IP %s is the same as host IP, they must be different"
+	tooSmallDiskTemplate                    = "Disk is too small (disk only has %s, but %s are required)"
+	wrongDriveTypeTemplate                  = "Drive type is %s, it must be one of %s."
+	wrongMultipathTypeTemplate              = "Multipath device has path of type %s, it must be %s"
+	wrongISCSINetworkTemplate               = "iSCSI host IP %s is the same as host IP, they must be different"
+	ErrsInIscsiDisableMultipathInstallation = "Installation on multipath device is not possible due to errors on at least one iSCSI disk"
```

Comment:
```suggestion
	errsInIscsiDisableMultipathInstallation = "Installation on multipath device is not possible due to errors on at least one iSCSI disk"
```
I don't think we want to make that variable public

Reply:
I also use it in internal/host. Do you think it would be better to place this constant there as well instead of making it public?

Reply:
ah I missed that, it's fine them